README.md

```diff
@@ -144,6 +150,18 @@ All commands should be run from `workbench/api`
 
 While the API is running locally, saving a .java file should cause a recompile and reload of that class. Status is logged to the console. Not all changes reload correctly (e.g., model classes do not appear to reload more than once).
 
+#### Caveats
+
+The first time a server is started after a database clear (`./project.rb docker-clean`, `./project.rb run-local-all-migrations`), it may take some time for a billing project to become available for workspace creation. Generally, this takes about 5 minutes, but has been known to take upwards of 40 in some cases. The billing project buffer size is set to 2 for local environments (`config_local.json`). A cron job runs every minute while the API server is running to check if there are empty slots.
```

Comment:
does `./project.rb run-local-all-migrations` drop your database? I think it might just run the schema updates.

Reply:
It's `docker-clean` that drops the DB